app/segments/drill/[id]/submission/input.js

```diff
@@ -349,7 +405,7 @@ export default function Input({ drillInfo, setToggleResult, setOutputData }) {
             );
 
             setOutputData(outputData);
-            uploadAttempt(outputData);
+            uploadAttempt(outputData, auth["currentUserId"], auth["currentTeamId"], assigned_time, id, queryClient);
```

Comment:
separate auth out into currentUserId instead of accessing them like this.
const { currentUserId, currentTeamId } = currentAuthContext();